app/services/claims_export_service.rb

```diff
@@ -67,6 +72,13 @@ def export_text_file(claim:, to:)
     stored_file.download_blob_to File.join(to, txt_fn)
   end
 
+  def export_claimants_text_file(claim:, to:)
+    stored_file = claim_export_service.new(claim).export_claimants_txt
+    primary_claimant = claim.primary_claimant
+    txt_fn = "#{claim.reference}_ET1a_#{primary_claimant.first_name.tr(' ', '_')}_#{primary_claimant.last_name}.txt"
```

Comment:
Should "ET1a" be lower case?

Reply:
No, thats right - this is exporting from our system to ATOS which wants them like this